<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>**********************************************************************
 **
 ** Copyright Â© 2011, Jason Gedge &lt;gedge -at- ualberta -dot- ca&gt;
 **
 ** This file is part of StereoReconstruction.
 **
 ** StereoReconstruction is free software: you can redistribute it and/or
 ** modify it under the terms of the GNU General Public License as published
 ** by the Free Software Foundation, either version 3 of the License, or
 ** (at your option) any later version.
 **
 ** StereoReconstruction is distributed in the hope that it will be
 ** useful, but WITHOUT ANY WARRANTY; without even the implied warranty
 ** of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 ** GNU General Public License for more details.
 **
 ** You should have received a copy of the GNU General Public License along
 ** with StereoReconstruction. If not, see &lt;http:www.gnu.org/licenses/&gt;.
 **
 **********************************************************************</comment>
 <class>CalibrationSetupDialog</class>
 <widget class="QDialog" name="CalibrationSetupDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cameras:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Image Sets:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="camerasList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="imageSetsList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibrationSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibrationSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
